bigbluebutton-html5/imports/ui/components/actions-bar/actions-dropdown/component.jsx

```diff
@@ -77,6 +77,28 @@ class ActionsDropdown extends Component {
 
     if (!isUserPresenter) return null;
 
+    function ScreenshareButton(props) {
```

Comment:
I think this should be in a separate component.jsx and imported here

Reply:
I would insert it right into render.. instead of a current `<ScreenshareButton />` it can be:

```
render() {
  const { stopDesktopShareLabel, desktopShareLabel, stopDesktopShareDesc, desktopShareDesc } = intlMessages;

......

<DropdownListItem
  icon="desktop"
  label={intl.formatMessage(isVideoBroadcasting ? stopDesktopShareLabel : desktopShareLabel)}
  description={intl.formatMessage(isVideoBroadcasting ? stopDesktopShareDesc : desktopShareDesc)}
  onClick={isVideoBroadcasting ? handleUnshareScreen : handleShareScreen }
/>;
```

Reply:
I would insert it right into render.. instead of a current `<ScreenshareButton />` it can be:

```
render() {
  const { stopDesktopShareLabel, desktopShareLabel, stopDesktopShareDesc, desktopShareDesc } = intlMessages;

......

<DropdownListItem
  icon="desktop"
  label={intl.formatMessage(isVideoBroadcasting ? stopDesktopShareLabel : desktopShareLabel)}
  description={intl.formatMessage(isVideoBroadcasting ? stopDesktopShareDesc : desktopShareDesc)}
  onClick={isVideoBroadcasting ? handleUnshareScreen : handleShareScreen }
/>;
```